### ISSUE TITLE
Requirements conflict with Home Assistant Discord notification integration

### ISSUE THREAD

**Issue by xiidoz** [LordBoos/discord_game#6]
Applies to:
- discord_game v. 3.0.2
- Home Assistant v. 104.2 (and previous releases)

If you are using the [Home Assistant Discord integration for notifications](https://www.home-assistant.io/integrations/discord/), discord.py v. 1.2.5 gets loaded, even though discord_game requires a newer version of discord.py.

Until the new version is officially released and the Home Assistant integration is updated, the following workaround can be applied:

Create a folder ```discord``` under ```custom_components``` and copy the files from https://github.com/home-assistant/home-assistant/tree/dev/homeassistant/components/discord in that folder. Update the manifest.json to use the newer version of discord.py, so that the file looks like this:
```json
{
  "domain": "discord",
  "name": "Discord",
  "documentation": "https://www.home-assistant.io/integrations/discord",
  "requirements": ["git+https://github.com/Rapptz/discord.py@23f16618fdf5fcbeeefed7acb56b579e74bc88d8#discord.py==1.6.0"],
  "dependencies": [],
  "codeowners": []
}
```

Restart Home Assistant and both Discord integrations should be working as intended.

**Comment by LordBoos** [comment]
Can you please test the new version 3.0.3? Discord.py library update was released. Official component still uses 1.2.5, but it might work. If not, the official component will have new Discord.py library in the next version, as it was already merged to their code: https://github.com/home-assistant/home-assistant/pull/31391

**Comment by xiidoz** [comment]
It does work under home assistant dev branch without any workaround. We can close this Issue as soon as home-assistant/home-assistant#31391 gets merged to [master](https://github.com/home-assistant/home-assistant/tree/master).

⚠ The version conflict between home assistant's current release (0.104.4) still exists, as home assistant force loads discord.py v. 1.2.5, which is incompatible with discord_game v. 3.0.2 and 3.0.3

**Comment by xiidoz** [comment]
Is now solved with home assistant 0.106.0.
